File: lib/app/models/establishment.dart
import 'package:pub/app/models/room.dart';
class Establishment {
  late String _name;
  late String _latitude;
  late String _longitude;
  late Room _room;

  Establishment();

  Establishment.with_JSON (Map json) :
      _name = json['nome'],
      _latitude = json['latit'],
     _longitude = json['longit'];

  //GETTERS
  get getName => _name;
  get getLatitude => _latitude;
  get getLongitude => _longitude;
  get getRoom => _room;

//SETTERS
  setName(String nome) => _name = nome;
  setLatitude(String latitude) => _latitude = latitude;
  setLongitude(String longitude) => _longitude = longitude;
  setRoom(Room room) => _room = room;
}

File: pub-node/src/models/establishment.ts
export class Establishment{
    constructor(public nameConstructor:string,public latitudeConstructor:number,public longitudeConstructor:number){
        this.name = nameConstructor;
        this.latitude = latitudeConstructor;
        this.longitude = longitudeConstructor;
    }
    name:string;
    latitude:number;
    longitude:number;
}